nbdev/process.py

```diff
@@ -76,7 +76,7 @@ def instantiate(x, **kwargs):
     "Instantiate `x` if it's a type"
     return x(**kwargs) if isinstance(x,type) else x
 
-def _mk_procs(procs, nb): return L(procs).map(instantiate, nb=nb)
+def _mk_procs(procs, nb, path): return L(procs).map(instantiate, nb=nb, path=path)
```

Comment:
AFAICT none of the changes in this file are needed - you can get the path from `self.nb.path_`

Reply:
Done (also thanks for explaining the bit about `execnb.read_nb`)